compose/README.md

```diff
@@ -1,100 +1,49 @@
-4. Specify Gradle JVM to use JDK 11 in InteliJ IDEA Preferences (`Build, Execution, Deployment -> Build Tools -> Gradle`)
+## Skiko
 
-## Scripts
-Publish artifacts to the local directory `out/androidx/build/support_repo/org/jetbrains/compose`:
-```
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom
-./scripts/publish
-```
-(on Windows it doesn't build at the moment, the issue is in the line `packageInspector(project, project(":compose:ui:ui-inspection")` in `compose/frameworks/support/compose/ui/ui/build.gradle`)
+Compose Multiplatform uses [Skiko](https://github.com/JetBrains/skiko). A low-level library that hides platform
+complexity and provides a simple interface for rendering, event handling, window management, and other features. Skiko
+uses Skia as graphical API.
 
-Publish extended icons:
-```bash
-./scripts/publishExtendedIcons
-```
+## Other parts
 
-### Run tests
+- [Gradle Plugin](https://github.com/JetBrains/compose-multiplatform/tree/master/gradle-plugins)
+- [IDEA Plugin](https://github.com/JetBrains/compose-multiplatform/tree/master/idea-plugin)
+- [Examples](https://github.com/JetBrains/compose-multiplatform/tree/master/examples)
 
-Run tests for Desktop:
-```bash
-./scripts/testDesktop
-```
+## Publishing
 
-Run tests for Web:
-```bash
-./scripts/testWeb
-```
+Compose Multiplatform libraries can be published to local Maven with the following steps:
 
-Run tests for UIKit:
-```bash
-./scripts/testUIKit
-```
+1. Set `COMPOSE_CUSTOM_VERSION` environment variable
 
-Run tests for Android:
 ```bash
-./scripts/testAndroidUnit.sh
-```
-```bash
-./scripts/testAndroidInEmulator.sh # first run android simulator or connect device
-```
-
-### Run samples
-Run jvm desktop sample:
-```bash
-./scripts/runGradle run
-```
-another jvm desktop samples: 
-```bash
-./scripts/runGradle run1 run2 run3 runSwing runWindowApi runVsync runLayout
-```
-
-Run wasm sample:
-```bash
-./scripts/runGradle runMppJs
+export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version
 ```
 
-Run native macos sample:
-```bash
-./scripts/runGradle runMppMacos
-```
-
-## Multiplatform build to mavenLocal
+2. Publish core libraries
+   using [instructions](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/MULTIPLATFORM.md)
+3. Publish Gradle plugin
 
 ```bash
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version &&\
-./scripts/publishToMavenLocal -Pcompose.platforms=all &&\
 ./scripts/publishGradlePluginToMavenLocal
 ```
-`-Pcompose.platforms=all` could be replace with comma-separated list of platforms, such as `js,jvm,androidDebug,androidRelease,macosx64,uikit`.
 
-## Publish additional libraries to mavenLocal
+4. Publish additional components
 ```bash
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version &&\
-./scripts/publishWebComponentsToMavenLocal &&\
-./scripts/publishExtendedIconsToMavenLocal -Pcompose.platforms=all &&\
 ./scripts/publishComponentsToMavenLocal
 ```
+
+5. Publish additional Web components
```

Comment:
Let's call it Compose HTML library here.

Also the script can be renamed.
Plus, the script itself needs to be updated:
https://github.com/JetBrains/compose-multiplatform/blob/master/compose/scripts/publishWebComponentsToMavenLocal

`pushd ../../web` -> `pushd ../../html`

If it's okay to do this change in this PR

Reply:
I think it is ok, will do!

Reply:
Done